.gitignore

```
@@ -2,3 +2,6 @@ __pycache__
 .tox/
 log.txt
 .idea/
+node_modules/
+package.json
+package-lock.json
```

nit: Ending a file with a new line prevents the last line from appearing in future commits as a change.
```suggestion
package-lock.json

```